CoreNetKit/Core/ServerPart/BaseCoreServerRequest.swift

```diff
@@ -157,6 +160,10 @@ extension BaseCoreServerRequest {
         case failure(CoreServerResponse)
     }
 
+    func createNoneParamRequest() -> DataRequest {
```

Comment:
hmmm. Ia it worth it? 
``Swift let request = self.reateSingleParamRequest(nil)` It would be enough

Reply:
I think it looks good. `createNoneParamRequest` can be a place to add some logic for requests without parameters, if we will need it in future.

Reply:
What logic? for example?

Reply:
Anyway after the refactoring everything will be different ...